portfolio/src/main/webapp/script.js

```diff
@@ -16,12 +16,41 @@
  * Cycles through greetings in different languages for front page. 
  */
 const greetings =['Hi, nice to meet you!', '¡Hola, encantado de conocerte!', '你好，很高興見到你', 'Salut! Enchanté!'];
-var textElement = document.getElementById('changeText');
+let textElement = document.getElementById('changeText');
 textElement.innerHTML = greetings[0];
-var counter = 1;
-var intervalLength = setInterval(change, 2000);
+let counter = 1;
+setInterval(change, 2000);
 document.getElementById('commentBtn').addEventListener('click', getComments);
 
+function initMap() {
+    const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422403, lng: -122.088073}, zoom: 5});
+    const titles = ['Home', 'UMass Amherst', 'Georgia Tech', 'Home Away From Home', 'An Extraordinary Layover'];
+    const lats = [37.3230, 42.3868, 33.7756, 19.4564, 51.5074];
+    const lngs = [-122.0322,-72.5301, -84.3963, 72.7925, -0.1278]
+    const descriptions = ['Where I currently live, and spent my high school years. Home to Apple HQ, the best boba, \
+    and my closest friends :)', 'Snowy, remote, but cozy; where I spent my first 1.5 years of college.', 'Where I \
+    currently attend college!', 'Where my parents grew up; the comfort of my extended family and the cities unique \
+    culture always brings me back', 'Spent an ambitious 8 hours searching for all the famous landmarks, check out pictures in misc!'];
+    for (let i = 0; i < titles.length; i++) {
+        addLandmark(map, lats[i], lngs[i], titles[i], descriptions[i])
+    }
```

Comment:
I think have these all be separate arrays makes it a bit hard to parse which titles match with which lats/lngs/descriptions.

What about add a new class, something like:
```
class Landmark {
  constructor(title, lat, lng, description) {
      this.title = title;
      this.lat = lat;
      this.lng = lng;
      this.description = description;
  }
}
```

And then you can create new Landmarks with: `new Landmark('Home', 37.3230, -122.0322, 'Where I..')` and put all your Landmarks into a single array that you iterate through. Then, you can also change `addLandmark` to take in just two params: `function addLandmark(map, landmark)`